web/src/components/DisablePasswordLoginDialog.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import { toast } from "react-hot-toast";
+import { useTranslate } from "@/utils/i18n";
+import { useGlobalStore } from "@/store/module";
+import * as api from "@/helpers/api";
+import Icon from "./Icon";
+import { generateDialog } from "./Dialog";
+import { Button } from "@mui/joy";
```

Comment:
Imports for third-party dependencies should be placed above the component. We can put it on the first line of the file.

Reply:
Wasn't aware about that convention. I hope this is correct now?

https://github.com/usememos/memos/blob/ae2856c5da2049139fd0652273e130f4ba636f97/web/src/components/DisablePasswordLoginDialog.tsx#L1-L8